API.md

```diff
@@ -241,8 +241,8 @@ Drag and drop SVG, HTML or Canvas using mouse or touch input.
 * [*drag*.clickDistance](https://github.com/d3/d3-drag/blob/master/README.md#drag_clickDistance) - set the click distance threshold.
 * [*drag*.on](https://github.com/d3/d3-drag/blob/master/README.md#drag_on) - listen for drag events.
 * [*event*.on](https://github.com/d3/d3-drag/blob/master/README.md#event_on) - listen for drag events on the current gesture.
-* [d3.dragDisable](https://github.com/d3/d3-drag/blob/master/README.md#dragDisable) -
-* [d3.dragEnable](https://github.com/d3/d3-drag/blob/master/README.md#dragEnable) -
+* [d3.dragDisable](https://github.com/d3/d3-drag/blob/master/README.md#dragDisable) - prevents native drag-and-drop and text selection on the specified window.
```

Comment:
Nice additions! A few notes:

 * The "on the specified window" doesn't really make sense, without the context of the argument. Maybe best to leave it out? "prevents native drag-and-drop and text selection."
 * The other blurbs use things like "set" and "listen", rather than "sets" and "listens", so for consistency, I'd suggest making it start with "prevent".

So my final suggested wording would be:

 * "prevent native drag-and-drop and text selection"
 * "enable native drag-and-drop and text selection"